deploy/templates/default-azuredeploy-webjobs.json

```diff
@@ -0,0 +1,604 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
>$schema [](start = 3, length = 7)

We can have a separate template for now but ideally I would like to collapse this and others into one template with conditionals.  We have many *different* flavors of ARM templates today and they are becoming harder to manage and keep in sync with changes.